### DISCUSSION TITLE
Arm64 error when installing Emgu.CV.runtime.windows

### DISCUSSION THREAD

**Discussion by helderc** [emgucv/emgucv#861] [Q&A]
Hi all.
I'm creating an image in Mat format with the following code:

```
_mtx = new Mat(pixelData.Height, 
                           pixelData.Width,
            Emgu.CV.CvEnum.DepthType.Cv32F, 
                        1);
```
When running the code throws the exception: 
`System.TypeInitializationException: 'The type initializer for 'Emgu.CV.MatInvoke' threw an exception.'
DllNotFoundException: Unable to load DLL 'cvextern': The specified module could not be found. (Exception from HRESULT: 0x8007007E)`

According to this [1] thread, it seems that I need to install the `Emgu.CV.runtime.windows`. However, when I try to install it I got the error related to the arm64 package:

`Severity	Code	Description	Project	File	Line	Suppression State
Error		Could not install package 'Emgu.runtime.windows.msvc.rt.arm64 19.35.32217.1'. You are trying to install this package into a project that targets '.NETFramework,Version=v4.8', but the package does not contain any assembly references or content files that are compatible with that framework. For more information, contact the package author.				
`

[1] https://stackoverflow.com/questions/8028523/unable-to-load-cvextern-in-a-c-sharp-project

How can I solve this? Do I need to have this arm64?
Thank you in advance.

**Comment by helderc** [comment]
Just for future reference, it seems the problem was related to the "Any CPU" targeting architecture.

This post [1] was very helpful to solve it.

[1] https://github.com/emgucv/emgucv/discussions/823

Thank you all.

**Comment by emgucv** [comment]
Detail instruction can be found here as well:
https://www.emgu.com/wiki/index.php/Download_And_Installation#Targeting_.Net_Framework

Seems like the issue has been resolved. Closing it now.
